hack/build.sh

```diff
@@ -19,10 +19,11 @@ set -x
 # 1) Make a release-NN branch as normal.
 # 2) Update VERSIONS below (on main) to include the new version.
 #    Order matters :-), Most recent first.
-VERSIONS=("0.24" "0.23" "0.22" "0.21")                  # Docs version, results in the url e.g. knative.dev/docs-0.23/..
-RELEASE_BRANCHES=("v0.24.0")                     # Release version for serving/eventing yaml files and api references.
+VERSIONS=("0.25" "0.24" "0.23" "0.22")                  # Docs version, results in the url e.g. knative.dev/docs-0.23/..
+VERSIONS_GENERATORS=("mkdocs" "mkdocs" "hugo" "hugo")  # update this to always be 4 in the next two releases replace hugo with mkdocs, remove the copy of static hugo site at the bottom
+RELEASE_BRANCHES=("v0.25.0")                     # Release version for serving/eventing yaml files and api references.
```

Comment:
```suggestion
RELEASE_BRANCHES=("v0.25.0" "v0.24.0")                     # Release version for serving/eventing yaml files and api references.
```

Reply:
I think this will fix both the lack of a version number in the banner and the lack of reference docs actually.

Reply:
Good catch, thank you for the review. It looks like we are not catching an error when we download the API doc, adding `-L` to handle the redirect to a nonexisting endpoint.

Reply:
ok verified now with `-L` we will catch the error like in this netlify build error
https://app.netlify.com/sites/knative/deploys/6123d10ead5dde00081754bf?utm_source=github&utm_campaign=bot_dl#L228-L240